src/main/plugins/org.dita.pdf2/xsl/fo/loc.xsl

```diff
@@ -0,0 +1,256 @@
+                    <xsl:value-of select="$i"/>
+                  </fo:block>
+                  <fo:block>
+                    <xsl:value-of select="$bookReleaseDateString"/>
+                    <xsl:value-of select="$colonSymbol"/>
+                    <xsl:value-of select="$separator"/>
+                    <xsl:value-of select="format-date($timeRangeEndAsDate,'[MNn] [D], [Y]')"/>
+                  </fo:block>
+                </fo:block>
+              </fo:table-cell>
+            </fo:table-row>
+            <!-- Iterate over all change-items -->
+            <xsl:for-each select="1 to (count($changeset))">
+              <xsl:variable name="j" select="position()"/>
+              <xsl:variable name="currentChangeItem" select="$changeset[$j]"/>
+              <xsl:variable name="currentChangeItemChangeCompleted" select="$changeset[$j]/*[contains (@class, ' relmgmt-d/change-completed ')]"/>
+              <xsl:if test="$timeRangeStart != ''
+                            and $currentChangeItemChangeCompleted != ''
+                            and $timeRangeEnd">
+                <xsl:variable name="currentChangeItemChangeCompletedAsDate" select="xs:date($currentChangeItemChangeCompleted)"/>
+                <xsl:if test="($timeRangeStartAsDate &lt;= $currentChangeItemChangeCompletedAsDate) and ($currentChangeItemChangeCompletedAsDate &lt;= $timeRangeEndAsDate)">
+                  <fo:table-row xsl:use-attribute-sets="releaseManagementTable.changeItem.row">
+                    <fo:table-cell xsl:use-attribute-sets="releaseManagementTable.changeItem.cell">
+                      <fo:block xsl:use-attribute-sets="releaseManagementTable.changeItem.content">
+                        <xsl:call-template name="change-item">
+                          <xsl:with-param name="change-item" select="$currentChangeItem"/>
+                        </xsl:call-template>
+                      </fo:block>
+                    </fo:table-cell>
+                  </fo:table-row>
+                </xsl:if>
+              </xsl:if>
+            </xsl:for-each>
+          </fo:table-body>
+        </fo:table>
+      </fo:block>
+    </xsl:for-each>
+  </xsl:template>
+
+  <xsl:template match="*[contains(@class, ' relmgmt-d/change-item ')]">
+    <xsl:apply-templates select="*[contains(@class,' ditaot-d/ditaval-startprop ')]" mode="outofline"/>
+    <fo:list-block xsl:use-attribute-sets="ul">
+      <xsl:call-template name="commonattributes"/>
+      <xsl:apply-templates/>
+    </fo:list-block>
+    <xsl:apply-templates select="*[contains(@class,' ditaot-d/ditaval-endprop ')]" mode="outofline"/>
+  </xsl:template>
+
+  <xsl:template match="*[contains(@class, ' relmgmt-d/change-item ')]/*" priority="10"/>
+
+  <xsl:template name="change-item">
+    <xsl:param name="change-item"/>
+    <xsl:for-each select="$change-item/*[contains (@class, ' relmgmt-d/change-organization ')][1]">
+      <xsl:call-template name="change-organization">

```

Comment:
Instead of named templates, use templates with match. Either with modes or preferably just based on the element that is matched.